test/fork/SNstCalls.t.sol

```diff
@@ -5,29 +5,29 @@ import "test/fork/ForkTestBase.t.sol";
 
 contract EthereumControllerSwapNSTToSNSTFailureTests is ForkTestBase {
 
-    function test_swapNSTToSNST_notRelayer() external {
+    function test_depositToSNST_notRelayer() external {
         vm.expectRevert(abi.encodeWithSignature(
             "AccessControlUnauthorizedAccount(address,bytes32)",
             address(this),
             RELAYER
         ));
-        ethereumController.swapNSTToSNST(1e18);
+        ethereumController.depositToSNST(1e18);
     }
 
-    function test_swapNSTToSNST_frozen() external {
+    function test_depositToSNST_frozen() external {
         vm.prank(freezer);
         ethereumController.freeze();
 
         vm.prank(relayer);
         vm.expectRevert("EthereumController/not-active");
-        ethereumController.swapNSTToSNST(1e18);
+        ethereumController.depositToSNST(1e18);
     }
 
 }
 
 contract EthereumControllerSwapNSTToSNSTTests is ForkTestBase {
```

Comment:
Name should be changed to `EthereumControllerDepositToSNSTTests`.

Reply:
Fixed